lansa-vmss-windows-autoscale-sql-database/azuredeploy.parameters.json

```diff
@@ -2,17 +2,62 @@
     "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
     "contentVersion": "1.0.0.0",
     "parameters": {
-        "vmssName": {
+        "stackName": {
             "value": "GEN-UNIQUE-10"
         },
+        "certificateBase64Encoded": {
+            "value" : "GEN-SELFSIGNED-CERT-PFXDATA"
+        },
+        "certificatePassword": {
+            "value": "GEN-SELFSIGNED-CERT-PASSWORD"
+        },
+        "databaseNewOrExisting": {
+            "value": "new"
+        },
+        "databaseServerName": {
+            "value": "GEN-UNIQUE-10"
+        },
+        "databaseName": {
```

Comment:
move this to a defaultValue in the template

Reply:
Done